src/js/app.js

```diff
@@ -0,0 +1,113 @@
+const $todoInput = document.querySelector("#new-todo-title");
+const $toggleInput = document.querySelector(`#todo-list`);
+const $todoCount = document.querySelector(`span strong`);
+
+$todoInput.addEventListener("keyup", addTodoItem);
+
+document.addEventListener("click", activeTodoItem);
+document.addEventListener("click", allTodoItem);
+document.addEventListener("click", finishTodoItem);
+
+function renderTodoItemTemplate(title) {
+    return ` <li class="active">
+                  <div class="view">
+                      <input class="toggle" onclick="onToggleTodoItem(event)" type="checkbox" >
+                      <label class="label" ondblclick="updateTodoItem(event)">${title}</label>
+                      <button class="destroy" onclick="deleteTodoItem(event)"></button>
+                  </div>
+                  <input class="edit" value=${title} />
+              </li>`;
+}
+
+function addTodoItem(event) {
+    const todoTitle = event.target.value;
+    const todoList = document.getElementById("todo-list");
+    if ("Enter" === event.key && "" !== todoTitle) {
+        todoList.insertAdjacentHTML("beforeend", renderTodoItemTemplate(todoTitle));
+        event.target.value = "";
+        count();
+    }
+}
+
+function onToggleTodoItem(event) {
+    const target = event.target;
+    if ("toggle" === target.className) {
```

Comment:
```suggestion
    if ("toggle" !== target.className) {
      return;
    }
```

if 문 내에 if 문을 중첩시키는 것 보다 early return 하는 것이 가독성이 더 좋을것 같아요!